service-tests/php-v-packagist.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'php', title: 'PHP version from Packagist' });
+module.exports = t;
+
+t.create('gets the package version of symfony')
+  .get('/v/packagist/symfony/symfony/dev-master.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'PHP', value: '^7.1.3' }));
+
+t.create('gets the package version of symfony 2.8')
+  .get('/v/packagist/symfony/symfony/v2.8.0.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'PHP', value: '>=5.3.9' }));
```

Comment:
Since these are both literals, this can be simplified like the one below.

Reply:
Here we are attached to specific [version/tag](https://github.com/symfony/symfony/releases/tag/v2.8.0) that will not change.

Reply:
Instead of

```
.expectJSONTypes(Joi.object().keys({ name: 'PHP', value: '>=5.3.9' }));
```

you can write

```
.expectJSON({ name: 'PHP', value: '>=5.3.9' });
```